src/components/CardGame/styles.module.scss

```diff
@@ -0,0 +1,7 @@
+.container {
```

Comment:
porque .module?

Reply:
É pra usar como nome de classe única veja de acordo com a [documentação do CRA](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet)
_"CSS Modules let you use the same CSS class name in different files without worrying about naming clashes. Learn more about CSS Modules"_